src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -204,147 +155,66 @@ export const BaseComment = ({
-                                        <MenuItem
-                                            className="bcs-Comment-unresolveComment"
-                                            data-resin-target={ACTIVITY_TARGETS.COMMENT_OPTIONS_EDIT}
-                                            data-testid="unresolve-comment"
-                                            onClick={() => handleStatusUpdate(COMMENT_STATUS_OPEN)}
-                                        >
-                                            <X16 />
-                                            <FormattedMessage {...messages.commentUnresolveMenuItem} />
-                                        </MenuItem>
-                                    )}
-                                    {canResolve && !isResolved && (
-                                        <MenuItem
-                                            data-resin-target={ACTIVITY_TARGETS.COMMENT_OPTIONS_EDIT}
-                                            data-testid="resolve-comment"
-                                            onClick={() => handleStatusUpdate(COMMENT_STATUS_RESOLVED)}
-                                        >
-                                            <Checkmark16 />
-                                            <FormattedMessage {...messages.commentResolveMenuItem} />
-                                        </MenuItem>
-                                    )}
-                                    {canEdit && (
-                                        <MenuItem
-                                            data-resin-target={ACTIVITY_TARGETS.COMMENT_OPTIONS_EDIT}
-                                            data-testid="edit-comment"
-                                            onClick={handleEditClick}
-                                        >
-                                            <Pencil16 />
-                                            <FormattedMessage {...messages.commentEditMenuItem} />
-                                        </MenuItem>
-                                    )}
-                                    {canDelete && (
-                                        <MenuItem
-                                            data-resin-target={ACTIVITY_TARGETS.COMMENT_OPTIONS_DELETE}
-                                            data-testid="delete-comment"
-                                            onClick={handleDeleteClick}
-                                        >
-                                            <Trash16 />
-                                            <FormattedMessage {...messages.commentDeleteMenuItem} />
-                                        </MenuItem>
-                                    )}
-                                </Media.Menu>
+                <div className="bcs-BaseComment-Header">
+                    <BaseCommentMetadata
+                        annotationActivityLink={annotationActivityLink}
+                        created_at={created_at}
```

Comment:
We should prefer camelCase for prop names.

Reply:
Indeed. I'm not 100% sure where this originates, but it's pretty entrenched. Search for `created_at` yielded 243 results. Where do you recommend that I transform this? Just on the new components? Or at BaseComment? Or some parent of BaseComment. Jlmk :)

Reply:
@jcamden the prop itself could be createdAt. The value doesn't though. We've used that other places in the code.

Reply:
Right, just wondering, if we do refactor the prop name, where should we do the transform? Should we refactor the pre-existing prop in BaseComment? Should I only transform it in the new component that I created? Or should I transform at the parent of BaseComment or some other ancestor?

Reply:
My many questions about this probably reflect the fact that I lean towards keep the prop name consistent as it's being drilled. Imo, it's better to change it everywhere than the have suddenly transform at some arbitrary point.

Reply:
I see your point. This is based on a type coming off of BaseFeedItem ready that was already there. One difference is that I don't think it's ever explicitly passed to a component. It's just  extracted via passed down props for various operations. I think in this case it makes sense to stay consistent with our conventions and just use createdAt={created_at}